### DISCUSSION TITLE
CS50 repository connection CS50.dev -> GitHub

### DISCUSSION THREAD

**Discussion by ghost** [community/community#129062] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

VSCode

### Body

Hello Community, 

I'm currently doing the CS50 Harvard Programming Course and I'm struggeling at the beginning. 

CS50.dev is connected to a repository 3380749 [codespaces: curly waffle] on GitHub. 

I also connected GitHub to CS50 for the upload of the different tasks (skripts). A repository me50/MY_USERNAMEXY is displayed under my repositories. I although uploaded the first test (Scratch skript).

So on GitHub it's seems like I have to repositories: 
-code50/33804479) connected to tthe web version of vsstudio 
-me50/MY_USERNAMEXY were I uploaded the first task, which I can't access in the CS50.dev IDE (web based)

When I go to CS50.dev the repository 3380749 is displayed, not the me50/MY_USERNAMEXY and if I enter submit50 the following message appears in the terminal: 

usage: submit50 [-h] [--logout] [--log-level {debug,info,warning,error}] [-V] slug
submit50: error: the following arguments are required: slug

I assume, that I connected the CS50.dev workspace with the wrong repository on github, but don't know how to change that if this is even what went wrong. 

Looking forward to some help, being sorry for my newbie question.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @Darnok93 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by asdacosta** [comment]
Ensure CS50.dev is connected to the correct GitHub repository (me50/MY_USERNAMEXY) where you've uploaded your assignments. Verify uploads are in the correct location. Use submit50 with the assignment slug (e.g., submit50 scratch) for submission.

**Reply by ebndev** [reply]
Hi @asdacosta, thanks for taking the time to answer!

@Darnok93 let us know if you have further questions and congrats on starting your coding journey 🌟
